contracts/Quest.sol

```diff
@@ -80,13 +78,13 @@ contract Quest is ReentrancyGuardUpgradeable, PausableUpgradeable, Ownable, IQue
         questId = questId_;
         questFee = questFee_;
         protocolFeeRecipient = protocolFeeRecipient_;
-        referralRewardFee = referralRewardFee_;
 
         // Setup default state
         questFactoryContract = IQuestFactory(payable(msg.sender));
         queued = true;
         referralClaimTotal = 0;
         totalReferralsFeesClaimed = 0;
+        referralRewardFee = 250; // 2.5%
```

Comment:
Why hardcode vs pass it in?

Reply:
We are going to hardcode it, but we could pass it in (and move hardcode to the QuestFactory instead)